iothub/service/src/Common/Exceptions/ErrorCode.cs

```diff
@@ -36,7 +36,21 @@ public enum ErrorCode
         DeviceInvalidResultCount = 400002,
 
         InvalidOperation = 400003,
+
```

Comment:
Yes, but it looks like you have some info to improve upon it. I'd suggest reactivating.

Also, be sure to remove `en-us` path from URLs.

Reply:
Also, as a note I've been claiming items in the tracking spreadsheet. You might check there to avoid duplicate work.